src/App.js

```diff
@@ -1,15 +1,24 @@
-
-import './App.css';
-import Greetings from './components/Greetings';
-import Counter from './components/Counter';
-import ListView from './components/ListView';
+import "./App.css";
```

Comment:
Adding previous exercise comments here
- use string literal in JSX
- use functional updates https://reactjs.org/docs/hooks-reference.html#functional-updates for setting state in the counter component.
- const [count, setCount] = useState(defaultValue) - fix this usage by referring to https://medium.com/@digruby/do-not-use-props-as-default-value-of-react-usestate-directly-818ee192f454
- need to use prettier for code formatting.